### PR TITLE
Script to set- up the test environment for the developers

### PR DESCRIPTION
**Pull request by MarcinGinszt**
Jira task: ISV-1209

### CODE REVIEW

File: ansible/inventory/group_vars/operator-pipeline.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ##### General settings
 env: unset
-oc_namespace: operator-pipeline-{{ env }}
+oc_namespace: "{{ (namespace) if (namespace) is defined else operator-pipeline-(env) }}"
```
Comment (Allda) [inline ansible/inventory/group_vars/operator-pipeline.yml:3]: If you pass the oc_namespace/namespace as env variable when running a playbook the value from inventory gets overwritten and you don't have to do this.

File: ansible/init-custom-env.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Creates service account for each operator-pipeline env; updates secret-vars.yml with token
+#
+# As arguments, it expects environments for which the sa should be created
+#
+# ./init.sh dev stage prod
+#       creates the sa for dev, stage and prod environments
```
Comment (MarcinGinszt) [inline ansible/init-custom-env.sh:8]: Just noticed the deprecated comment, updating it now

**Review by Allda — APPROVED** [review]
One minor detail, but otherwise it looks ok

File: ansible/inventory/group_vars/operator-pipeline.yml (hunk 1 above)
Comment (MarcinGinszt) [reply ansible/inventory/group_vars/operator-pipeline.yml]: Good point! Applied.

**Pull request merged**